### ISSUE TITLE
?. and ?? operators not handled by parser

### ISSUE THREAD

**Issue by korri123** [ben-sb/javascript-deobfuscator#59]
```js
var i = undefined;
i?.call();
var x = i ?? 0;
```
This will cause a parser error. Not sure if you use a library for this that needs updating or not.

**Comment by ben-sb** [comment]
Yes the parser is quite outdated unfortunately. Not too much I can do besides implementing the features myself/rewriting the project which I don't have time for currently.
